lighthouse-cli/performance-experiment/report/partials/left-nav.html

```diff
@@ -0,0 +1,70 @@
+<style>
+:root[data-report-context="perf-x"] .menu__link {
+  padding: 0 35px;
+}
+
+.menu__report-tab {
+  padding: 3px 13px;
+  color: #777;
+  border-top: 1px solid #EBEBEB;
+  display: flex;
+  flex-direction: column;
+  cursor: pointer;
+  text-decoration: none;
+  height: 55px;
+}
+
+.menu__report-tab:hover {
+  background-color: #448aff;
+  color: #FFF;
+}
+
+.menu__report-tab__url {
+  font-size: 15px;
+  font-weight: bold;
+  text-overflow: ellipsis;
+  line-height: 30px;
+  white-space: nowrap;
+  overflow: hidden;
+}
+
+.menu__report-tab__time {
+  font-size: 11px;
+  line-height: 11px;
+  padding-left: 2px;
+}
+</style>
+
+{{#createRelatedReportsContext}}
+  {{#each this}}
+    {{#if_less_than generatedTime ../../generatedTime }}
+    <a class="menu__report-tab" href="{{ reportUrl }}">
+      <p class="menu__report-tab__url" title="{{ url }}">Report for: {{ url }}</p>
+      <p class="menu__report-tab__time">Generated on: {{ format_time generatedTime }}</p>
+    </a>
+    {{/if_less_than}}
+  {{/each}}
+{{/createRelatedReportsContext}}
+<a class="menu__report-tab" href="#">
+ <p class="menu__report-tab__url" title="{{ url }}">Report for: {{ url }}</p>
```

Comment:
I tried this. It didn't fit very well imo. When I tested `http://localhost:8080`, the url here simply became `/.
Just to clarify, the URL in `Report for: {{ url }}` is the URL of the website being tested, not the report URL.
By the way, originally, the tabs in the left nav were meant to replace the meta data on the top bar. Since we are going to keep the top bar, shall we use report ID here?